docs/user-guide/cross-sections.ipynb

```diff
@@ -7,273 +7,203 @@
    "source": [
```

Comment:
<p>I like this more; thanks! Just some nitpicky thoughts:</p><p> </p><p>Rephrase: "...vertical dimension (i.e. levels) is present." </p><p> </p><p>Add: Continue the first sentence of second paragraph like this: </p><p>"... (i.e. levels) is present, temporal visualizations (e.g. Hovmöller diagram), and even visualizations of arbitrary dimensions along GCAs or constant lat/lon lines." </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/1321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>